zipline/finance/slippage.py

```diff
@@ -290,7 +291,7 @@ def process_order(self, data, order):
         )
 
 
-class MarketImpactBase(object):
+class MarketImpactBase(with_metaclass(ABCMeta)):
```

Comment:
What do you think of having this subclass `SlippageModel` instead of setting the metaclass, and then in `VolatilityVolumeShare` remove `FutureSlippageModel` as a parent class, since it already defines `allowed_asset_types = (Future,)`? That way this is more of a base slippage class than a mixin.